docs/skill_development/displaying-information/overview.md

```diff
@@ -1 +0,0 @@
-# Displaying information
```

Comment:
Should this just be removed if it's an empty file?

Reply:
<img width="1071" alt="image" src="https://github.com/NeonGeckoCom/neon-docs/assets/30268971/a7213001-fb1f-4ab1-a8e8-537f97abf57d">
I thought I did

Reply:
sorry, incomplete thought. It should probably be removed from [mkdocs.yaml](https://github.com/mikejgray/neon-docs/blob/650909df10fc8f52774e28a9c54cf4f137bf96be/mkdocs.yml#L91)